services/web/client/source/class/osparc/store/Store.js

```diff
@@ -210,6 +211,12 @@ qx.Class.define("osparc.store.Store", {
       if (idx !== -1) {
         studiesWStateCache[idx]["state"] = state;
       }
+      // Set the state as well in the current study, as a plaing property
+      const currentStudy = this.getCurrentStudy();
+      if (currentStudy && currentStudy.getUuid() === studyId) {
+        currentStudy.state = state;
+        this.fireDataEvent("changeCurrentStudy", currentStudy);
+      }
```

Comment:
I think this is added data that it doesn't belong to the study in the data model. There is no state column in the table project in the database right. I was afraid (happened to me once) that it would be sent to the server in some API calls, if added to the model, and that would break the openAPI validation..

Reply:
I meant Study data model in the frontend. That way, the Run/Stop button can just listen to the changes in that part of the model.

When serializing the study, you would have to make sure you get rid of the "state" part.

Reply:
I can do that... but I am not sure which one is better

Reply:
done